docs/pages/blog/2021.md

```diff
@@ -0,0 +1,185 @@
+
+We have achieved most of what we could have hoped for.
+
+### Company
+
+- The most important, we have welcomed 10 [members](/about/) in the company: Matheus, Michał, Jun, Danilo, Flavien, Benny, Alexandre, Jan, Bharat, Prakhar.
+- We have introduced a [public handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b) on how we run the company.
+  For example, we are using the OKRs work methodology to prioritize our efforts.
+- We held our first company [retreat](/blog/2021-q3-update/#retreat) 🏝 in Lisbon, Portugal 🇵🇹 , for members of the team that were not prevented by COVID-19 related travel restrictions, and who felt safe enough to travel.
+- We have [rebranded](/blog/material-ui-is-now-mui/) the company to a clean 3 letters acronym: mui.com. This move was key for us to grow beyond Material Design.
+
+### MUI Core
+
+- We have released [v5](/blog/mui-core-v5/), 21% of the npm downloads are on this major version now.
+- We have broken down the demos into smaller and focus on one aspect at a time.
+- We have polished and promoted most of the components that were in the lab to `@mui/material`.
+- We have introduced many new components:
+  - [Masonry](/components/masonry/)
+  - [Stack](/components/stack/)
+  - [Trap Focus](/components/trap-focus/)
+  - [Unstyled button](/components/buttons/#unstyled)
+  - [Unstyled slider](/components/slider/#unstyled)
+  - [Unstyled modal](/components/modal/#unstyled)
+  - [Unstyled slider](/components/switches/#unstyled)
+  - [Unstyled tabs](/components/tabs/#unstyled)
+
+### MUI X
+
+- We have released [v5](/blog/mui-x-v5/), 42% of the npm downloads are on this major version now.
+- We have introduced support for [custom locales](/components/data-grid/localization/).
+- We have introduced many new features while consolidating the existing ones:
+  - [Row editing](/components/data-grid/editing/#row-editing)
+  - [Cell editing](/components/data-grid/editing/#cell-editing)
+  - [Column pinning](/components/data-grid/columns/#column-pinning)
+  - [Tree data](/components/data-grid/group-pivot/#tree-data)
+  - [CSV export](/components/data-grid/export/#csv-export)
+  - [Print](/components/data-grid/export/#print)
+- We have grown the MIT downloads on npm and the number of licenses sold per month by a factor of x10 in 2021 compared to 2020.
+- We have grown the size of the team from 2 to 4 people (6 in January).
+
+## Looking at 2022
+
+I have been involved on MUI for 7 years and almost 3 years full-time now.
+It's amazing to see the progress the library has done [since then](https://v0.mui.com/#/components/app-bar).
+But at the same time, I'm astonished by how far we can still push the mission forward.
+My experience has been that the more we improve the library, the more we envision future ways to do it better.
+It's still [**day 1**](https://www.sec.gov/Archives/edgar/data/1018724/000119312517120198/d373368dex991.htm).
+
+What's our mission? We're making building great UIs more accessible.
```

Comment:
"making building"?

An suggestion to keep it more explicit:

```suggestion
What's our mission? We're making the task of building great UIs more accessible.
```

Reply:
```suggestion
What's our mission? To empower as many people as possible to build great UIs, faster.
```